rest-model/src/main/java/org/jboss/pnc/rest/restmodel/BuildEnvironmentRest.java

```diff
@@ -125,8 +125,8 @@ public String getImageId() {
         return systemImageId;
     }
 
-    public void setImageId(String imageId) {
-        this.systemImageId = imageId;
+    public void setImageId(String systemImageId) {
```

Comment:
I wouldn't think the parameter name here would have any effect.  Do the getter and setter method names also need to be changed from ImageId to SystemImageId?